src/main/java/br/com/caelum/clines/api/aircraftmodels/AircraftModelForm.java

```diff
@@ -0,0 +1,18 @@
+package br.com.caelum.clines.api.aircraftmodels;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+public class AircraftModelForm {
+    @NotBlank
+    private String description;
+
+    AircraftModelForm(String description) {
```

Comment:
What do you think about of using `@AllArgsConstructor`?

Reply:
`AircraftModelForm` is instantiated only by Spring representing the request body. We can add it anytime if needed.